swift/Tests/Blockchains/QtumTests.swift

```diff
@@ -61,14 +61,4 @@ class QtumTests: XCTestCase {
         XCTAssertEqual(BitcoinAddress(publicKey: xpubAddr2, prefix: P2PKHPrefix.qtum.rawValue).description, "QStYeAAfiYKxsABzY9yugHDpm5bsynYPqc")
         XCTAssertEqual(BitcoinAddress(publicKey: xpubAddr9, prefix: P2PKHPrefix.qtum.rawValue).description, "QfbKFChfhx1s4VXS9BzaVJgyKw5a1hnFg4")
     }
-
-    func testDeriveFromZPub() {
```

Comment:
I think we should keep bip84/zpub tests?

Reply:
In the current state they fail. Qtum defines its HDVersion as XPUB